README.md

```diff
@@ -1 +1,75 @@
 # weather-app
+![Linters](https://github.com/descholar-ceo/weather-app/workflows/Linters/badge.svg)
+
+## Project description
+This project is an implementation of a weather app.
+
+This project was done mainly for learning purposes, the aim was to practice using javascript async/await and working with APIs using fetch method. The project's specifications are listed on [The odin project](https://www.theodinproject.com/courses/javascript/lessons/weather-app)
+
+## Screenshot
+![demo picture](./src/assets/images/weather-app-screenshot.png)
+
+## Built with
+- HTML5
+- CSS3
+- Vanilla Javascript
+- Webpack
+
+## Features
+  - Search the weather info of any city on the earth
+  - Convert the gotten result from degrees celcius to fahreneit
+
+## Deployment
+- [Wearher app on netlify](https://weather-app-descholar.netlify.app)
+- [Wearher app on heroku](https://weather-app-descholar.heroku.com)
+
+## Running and testing it locally
+
+### Prerequisites
+- [NodeJS](https://nodejs.org/)
+- [Yarn](https://yarnpkg.com/) or just the default npm which comes with NodeJS
+
+### Setup
+- Run `git clone https://github.com/descholar-ceo/weather-app && cd weather-app` to get a copy of source codes on your local computer and to navigate inside the project directory
+- Run `yarn install`
+- Run `yarn build`
+- Create a file called `.env` add `PORT` in the same format of `.env.example`
```

Comment:
- [ ] In this instruction, it is not very clear on what should be done here. Kindly add that the port you want the webpack server to open the app on should be set here. e.g - `PORT=8080`. In the `.env.example`, what you have specified is just this - `PORT=`, and this is not well understood by non-techy users.

Reply:
Resolved